test/tests/services/filters-parser.test.js

```diff
@@ -118,27 +119,132 @@ describe('service > filters-parser', () => {
 
   describe('getParserForType', () => {
     describe('with a String type', () => {
-      it('should not try to cast it to ObjectId', () => {
+      it('should return the default parser', () => {
```

Comment:
Knowing I'm performing some code on the filter feature, I added some tests to ensure as possible I did not introduce regressions